### DISCUSSION TITLE
will termux work on HarmonyOS 2?

### DISCUSSION THREAD

**Discussion by varyaggit** [termux/termux-app#2413] [General]
Hi
will termux work on HarmonyOS 2 or not?
Would like to buy HUAWEI MatePad Pro 12.6 with HarmonyOS 2 but without termux this tablet is useless for me.
If somebody use termux on HarmonyOS 2 on this tablet or similar products of Huawei ?

**Comment by agnostic-apollo** [comment]
Apparently working. #2368

**Reply by varyaggit** [reply]
so termux is compatible with HarmonyOS 2.

**Reply by agnostic-apollo** [reply]
Termux is only officially compatible for Android OS and we currently only provide support for it currently. If HarmonyOS is compatible with Android OS apps as huawei says, then termux should work fine on it too. Some users are apparently using it as mentioned in the issue and haven't reported other issues, so probably is compatible. I can't say anything about the future though.

**Comment by varyaggit** [comment]
thanks for answer. I understood.
